README.md

```diff
@@ -1,43 +1,66 @@
 # Lua on Unikraft
 
-This application prints "hello world from initrd" using Lua.
+## KraftKit
 
-To configure, build and run this application you need to have [kraft](https://github.com/unikraft/kraft) installed.
-
-Configure the application:
-```
-kraft configure
+```console
+kraft build -t lua-qemu-x86_64-9pfs -j $(nproc)
+kraft build -t lua-qemu-x86_64-initrd -j $(nproc)
+kraft build -t lua-qemu-arm64-9pfs -j $(nproc)
+kraft build -t lua-qemu-arm64-initrd -j $(nproc)
+kraft build -t lua-fc-x86_64-initrd -j $(nproc)
+kraft build -t lua-fc-arm64-initrd -j $(nproc)
```

Comment:
I have one question: is there a particular reason for which the running of the application is done without `kraft run`?